src/test/java/ladder/PlayerTest.java

```diff
@@ -0,0 +1,52 @@
+package ladder;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import ladder.domain.Player;
+import ladder.domain.Players;
+
+public class PlayerTest {
+
+	@Test
+	@DisplayName("정상적인 플레이어 생성")
+	void generatePlayerTest() {
+		Player player = new Player("pobi");
+		assertThat(player).isEqualTo(new Player("pobi"));
+	}
+
+	@Test
+	@DisplayName("비정상적인 플레이어 생성 : 5자 초과")
+	void generatePlayerInvalidTest() {
+		assertThatIllegalArgumentException().isThrownBy(() -> {
+			new Player("pobi12");
+		});
+	}
```

Comment:
다음과 같이 hasMessage 를 통해 에러 메세지도 확인 해볼 수 있습니다! 😄 
```suggestion
	@Test
	@DisplayName("비정상적인 플레이어 생성 : 5자 초과")
	void generatePlayerInvalidTest() {
		assertThatIllegalArgumentException()
				.isThrownBy(() -> new Player("pobi12"))
				.withMessage("Player Name은 최대 5자입니다.");
	}
```